### DISCUSSION TITLE
Convert input.msh into into input.g

### DISCUSSION THREAD

**Discussion by IntePNum** [peridigm/peridigm#337] [General]
Hello, dear professors,

I have a trouble when convert the discreatzition files.

such as show in the attachment pictures.
![1](https://github.com/peridigm/peridigm/assets/108223279/0744613b-b8d6-4c2e-b760-3f1eb1057a20)
![2](https://github.com/peridigm/peridigm/assets/108223279/52d74003-c974-43a8-b92c-f4824153b7e3)

**Comment by oldninja** [comment]
It looks like this is a meshio problem... Tabs instead of spaces?

**Reply by IntePNum** [reply]
![Uploading image.png…]()
I change all the space to Tab, and found a "segment fault" error

**Reply by IntePNum** [reply]
such as this picture
![3](https://github.com/peridigm/peridigm/assets/108223279/30f02df4-06f5-4472-a018-76f1f5c89b12)

**Reply by oldninja** [reply]
Are you using the correct python environment? With all the dependencies?

**Reply by IntePNum** [reply]
Yes, I installed the python enviroment.
